src/main/java/com/limechain/network/protocol/grandpa/GrandpaProtocol.java

```diff
@@ -0,0 +1,74 @@
+package com.limechain.network.protocol.grandpa;
+
+import com.limechain.network.encoding.Leb128LengthFrameDecoder;
+import com.limechain.network.encoding.Leb128LengthFrameEncoder;
+import io.libp2p.core.Stream;
+import io.libp2p.protocol.ProtocolHandler;
+import io.libp2p.protocol.ProtocolMessageHandler;
+import io.netty.buffer.ByteBuf;
+import io.netty.handler.codec.bytes.ByteArrayEncoder;
+import lombok.extern.java.Log;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.concurrent.CompletableFuture;
+
+@Log
+public class GrandpaProtocol extends ProtocolHandler<GrandpaController> {
+    public static final int TRAFFIC_LIMIT = 1024 * 1024;
```

Comment:
I too prefer using 1 variable instead of 2 wherever possible, but it would be nice to update the other protocols as well. In `BlockAnounceProtocol` for example we have 2 separate fields holding the same value.